lib/messages/commands/reject.js

```diff
@@ -3,19 +3,65 @@
 var Message = require('../message');
 var inherits = require('util').inherits;
 var bitcore = require('bitcore');
-var BufferUtil = bitcore.util.buffer;
+var utils = require('../utils');
+var BufferReader = bitcore.encoding.BufferReader;
+var BufferWriter = bitcore.encoding.BufferWriter;
 
-// todo: add payload: https://en.bitcoin.it/wiki/Protocol_documentation#reject
+/**
+ * The reject message is sent when messages are rejected.
+ *
+ * @see https://en.bitcoin.it/wiki/Protocol_documentation#reject
+ * @param {Object=} arg - properties for the reject message
+ * @param {String=} arg.message - type of message rejected
+ * @param {Number=} arg.ccode - code relating to rejected message
+ * @param {String=} arg.reason - text version of reason for rejection
+ * @param {Buffer=} arg.data - Optional extra data provided by some errors.
+ * @param {Object} options
+ * @extends Message
+ * @constructor
+ */
 function RejectMessage(arg, options) {
+  if (!arg) {
+    arg = {};
+  }
   Message.call(this, options);
   this.command = 'reject';
+  this.message = arg.message;
+  this.ccode = arg.ccode;
+  this.reason = arg.reason;
+  this.data = arg.data;
 }
 inherits(RejectMessage, Message);
 
-RejectMessage.prototype.setPayload = function() {};
+RejectMessage.CCODE = {
+  REJECT_MALFORMED:       0x01,
```

Comment:
minor nit: spacing should be similar with the rest of the code base (e.g. https://github.com/bitpay/bitcore/blob/master/lib/crypto/signature.js#L307)